misc/extension_api_validation/4.4-stable.expected

```diff
@@ -321,3 +321,5 @@ Validate extension JSON: Error: Field 'classes/RichTextLabel/methods/add_image/a
 Validate extension JSON: Error: Field 'classes/RichTextLabel/methods/update_image/arguments': size changed value in new API, from 11 to 12.
 
 Optional argument added. Compatibility methods registered.
+
+Validate extension JSON: JSON file: Field was added in a way that breaks compatibility 'classes/Timer/properties/time_left': setter
```

Comment:
is this change ok? I mean, do I need to add any kind of format or versioning or something?

Reply:
See the above examples, should be:
```suggestion
GH-107920
---------

Validate extension JSON: JSON file: Field was added in a way that breaks compatibility 'classes/Timer/properties/time_left': setter
```

And add some description below